KafkaSaslSsl test fails on FIPS environment

### Bug description

Test KaffkaSaslSslTest fails on FIPS with

```
	Caused by: java.util.concurrent.ExecutionException: org.apache.kafka.common.errors.SaslAuthenticationException: Authentication failed during authentication due to invalid credentials with SASL mechanism SCRAM-SHA-512
		at org.apache.kafka.clients.producer.KafkaProducer$FutureFailure.<init>(KafkaProducer.java:1503)
		at org.apache.kafka.clients.producer.KafkaProducer.doSend(KafkaProducer.java:1041)
		at org.apache.kafka.clients.producer.KafkaProducer.send(KafkaProducer.java:943)
		at org.apache.kafka.clients.producer.KafkaProducer.send(KafkaProducer.java:825)
		at org.apache.camel.quarkus.kafka.sasl.KafkaSaslSslResource.post(KafkaSaslSslResource.java:65)
		at org.apache.camel.quarkus.kafka.sasl.KafkaSaslSslResource_ClientProxy.post(Unknown Source)
		at java.base/java.lang.reflect.Method.invoke(Method.java:568)
		at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:154)
		at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:118)
		at org.jboss.resteasy.core.ResourceMethodInvoker.internalInvokeOnTarget(ResourceMethodInvoker.java:560)
		at org.jboss.resteasy.core.ResourceMethodInvoker.invokeOnTargetAfterFilter(ResourceMethodInvoker.java:452)
		at org.jboss.resteasy.core.ResourceMethodInvoker.lambda$invokeOnTarget$2(ResourceMethodInvoker.java:413)
		at org.jboss.resteasy.core.interception.jaxrs.PreMatchContainerRequestContext.filter(PreMatchContainerRequestContext.java:321)
		at org.jboss.resteasy.core.ResourceMethodInvoker.invokeOnTarget(ResourceMethodInvoker.java:415)
		at org.jboss.resteasy.core.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:378)
		at org.jboss.resteasy.core.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:356)
		at org.jboss.resteasy.core.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:70)
		at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:429)
		... 17 more
	Caused by: org.apache.kafka.common.errors.SaslAuthenticationException: Authentication failed during authentication due to invalid credentials with SASL mechanism SCRAM-SHA-512
```

Problem is caused by https://github.com/apache/camel-quarkus/commit/184932632e3f11a221aeece70052a0c3b98fa3fe

--- a/integration-tests/kafka-sasl-ssl/src/test/java/org/apache/camel/quarkus/kafka/sasl/KafkaSaslSslTestResource.java
+++ b/integration-tests/kafka-sasl-ssl/src/test/java/org/apache/camel/quarkus/kafka/sasl/KafkaSaslSslTestResource.java
@@ -33,7 +33,7 @@ import static io.strimzi.test.container.StrimziZookeeperContainer.ZOOKEEPER_PORT
 public class KafkaSaslSslTestResource extends KafkaTestResource {
     static final String KAFKA_KEYSTORE_PASSWORD = "changeit";
     //min lengthe is 32 because of SCRAM-SHA-512
-    static final String ALICE_PASSWORD = "alice_changeit";
+    static final String ALICE_PASSWORD = "changeitchangeitchangeitchangeit";
     static final String KAFKA_HOSTNAME = "localhost";
     static final String CERTS_BASEDIR = "target/certs";
 

--- a/integration-tests/kafka-sasl-ssl/src/test/resources/config/kafka_server_jaas.conf
+++ b/integration-tests/kafka-sasl-ssl/src/test/resources/config/kafka_server_jaas.conf
@@ -1,4 +1,4 @@
 KafkaServer {
    org.apache.kafka.common.security.scram.ScramLoginModule required
-  user_alice="IuepUrtaAXpwgTy6TPmInRAUinlK2acQL";
+  user_alice="changeitchangeitchangeitchangeit";
 };

